stripe/src/test/java/com/stripe/android/view/CountryTextInputLayoutTest.kt

```diff
@@ -108,24 +109,29 @@ class CountryTextInputLayoutTest {
         countryTextInputLayout.setAllowedCountryCodes(setOf("fr", "de"))
         assertEquals(
             "FR",
-            countryTextInputLayout.selectedCountry?.code
+            countryTextInputLayout.selectedCountryCode?.twoLetters
         )
     }
 
     @Test
     fun validateCountry_withInvalidCountry_setsSelectedCountryToNull() {
-        assertNotNull(countryTextInputLayout.selectedCountry)
+        assertNotNull(countryTextInputLayout.selectedCountryCode)
         countryTextInputLayout.countryAutocomplete.setText("invalid country")
         countryTextInputLayout.validateCountry()
-        assertNull(countryTextInputLayout.selectedCountry)
+        assertNull(countryTextInputLayout.selectedCountryCode)
     }
 
     @Test
     fun validateCountry_withValidCountry_setsSelectedCountry() {
-        assertNotNull(countryTextInputLayout.selectedCountry)
+        assertNotNull(countryTextInputLayout.selectedCountryCode)
         countryTextInputLayout.countryAutocomplete.setText("Canada")
         countryTextInputLayout.validateCountry()
-        assertEquals("Canada", countryTextInputLayout.selectedCountry?.name)
+        countryTextInputLayout.selectedCountryCode?.let {
```

Comment:
Feels like we should use `requireNotNull` here to make it clear what we expect the value to be.